shared/lib/metrics/metrics.test.ts

```diff
@@ -76,4 +76,9 @@ describe('Metrics', function () {
     const predictions = [0.5, 0.5]
     assert.deepEqual(metrics.rocAucScore(labels, predictions), 0.5)
   })
+  it('empty input', function() {
```

Comment:
Add a space after the function keyword. `function()` -> `function ()`.

Apparently in Prettier.js version 2 and above this is the default. Here's the thread about it https://github.com/prettier/prettier/pull/3903. People have already argued about it extensively online but my opinion is that it doesn't really matter what it is as long as we all use the same version of Prettier.js.

My guess is that your local prettierjs version is still on the version 1, but all of the packages in this repo are on version 2+ (see package.json in `docs`, `scikitjs-browser`, and `scikitjs-node`)

Reply:
Thank you let me check and fix this!